pkgs/applications/misc/moz-phab/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, python3Packages, pkgs }:
```

```suggestion
{ lib, python3Packages}:
```
Please don't pass in pkgs as it is not very performant to override it.

---

should I also not pass `python3Packages` for the same reason?